rust/phoenix-channel/src/lib.rs

```diff
@@ -99,9 +100,18 @@ async fn create_and_connect_websocket(
         .await
         .map_err(|_| InternalError::Timeout { duration })??;
 
-    let (stream, _) = client_async_tls(make_request(url, user_agent)?, socket)
-        .await
-        .map_err(InternalError::WebSocket)?;
+    // Use rustls-platform-verifier to load native system certificates
+    let config = rustls::ClientConfig::with_platform_verifier();
```

Comment:
I assume we can now disable some other feature to get rid of the embedded certs?

Reply:
I don't think we need to do anything else, this is 1:1 what they recommend [in their example](https://github.com/rustls/rustls-platform-verifier). I presume that if you don't provide a config, it will use some default config that loads the internal certs. I haven't found a feature flag to get rid of the webpki-root dependency though.

Reply:
I believe it is declared here: https://github.com/firezone/firezone/blob/7730fdeda9d522fb1c7114b8716dbe5cb796e6d9/rust/phoenix-channel/Cargo.toml#L25

Reply:
Will do